src/pages/guides/using/securing_web_actions.md

```diff
@@ -26,7 +28,23 @@ If you fail in adding the authentication header or the secret is wrong, you will
 }
 ```
 
-## Non Web Action
+## Oauth (using the Adobe Identity Management System)
+Here is how you can enable IMS Authentication for a web action:
+
+```
+// create a web action with Require Validation on
+wsk action create my-requir-validation-web-action main.js --web true -a require-validation true
```

Comment:
We should not use `wsk cli` in our documentation.

Reply:
Right, but all documentation is with `wsk` now. I'll create another PR to change all occurrences to `aio`.

Reply:
But it's new documentation, by adding `wsk` commands we just increased our tech debt to replace it with the `aio`. I think we should follow the next approach: all new documentation should have only `aio`, changes in existing documentation should replace affected `wsk` by `aio`, and have a story/PR to replace `wsk` by `aio`.